proto/src/determined/trial/v1/trial.proto

```diff
@@ -320,3 +321,68 @@ message TrialSourceInfo {
   // Type for this trial_source_info
   TrialSourceInfoType trial_source_info_type = 5;
 }
+
+// Flat run respresentation.
+message Run {
+  // The id of the run.
+  int32 id = 1;
+  // The time the run was started.
+  google.protobuf.Timestamp start_time = 2;
+  // The time the run ended.
+  google.protobuf.Timestamp end_time = 3;
+  // The current state of the run(trial).
+  State state = 4;
+  // The tags of the run.
+  string tags = 5;
+  // The total size of checkpoints.
+  int64 checkpoint_size = 6;
+  // The count of checkpoints.
+  int32 checkpoint_count = 7;
+  // The type of searcher for the experiment.
+  string searcher_type = 8;
+  // The searcher metric name for the experiment.
+  string searcher_metric = 9;
+  // Signed searcher metrics value.
+  double searcher_metric_value = 10;
+  // The id of external run
+  int32 external_run_id = 11;
+  // Trial hyperparameters.
+  string hyperparameters = 12;
+  // summary metrics.
+  string metrics = 13;
+  // The id of the experiment linked to the run.
+  int32 experiment_id = 14;
+  // The id of the user who created the parent project.
+  int32 owner_id = 15;
+  // Original id of a forked or continued experiment.
+  google.protobuf.Int32Value forked_from = 16;
```

Comment:
what does forked_from mean for runs?

Reply:
Same as in the earlier comment we want the experiment fields returned but not shown by default.